extension/intents/navigation/navigation.toml

```diff
@@ -63,6 +63,7 @@ match = """
 description = "Navigate directly to a site, using Google's I'm Feeling Luck and the query"
 match = """
   (bring me | take me | go | navigate | show me | open) (to | find |) (page |) [query]
+  (bring me | take me | go | navigate | show me | open) (to | find |) (page |) (in |) (a |) (new |) [where] (and go | and) (to | find | open) [query]
```

Comment:
`[where]` here will catch anything, so it could also catch "open a new bucket to the new york times"

Adding a new type to `entityTypes.js` would fix this, though that would also be fine in a followup.